### PR TITLE
LPS-116840

### PR DESCRIPTION
**Pull request by RitaSilveira**
JIRA Ticket:
LPS-134149
LPS-116840

Motivation:
Test Automation for "Quick access to Content-related actions on the dashboard"

### CODE REVIEW

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/LPS-134149, cc: @CarlosBrichete

**Comment by RitaSilveira** [comment]
JIRA Ticket:
LPS-134149
LPS-116840

Motivation:
Test Automation for "Quick access to Content-related actions on the dashboard"

**Comment by CarlosBrichete** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -1287,6 +1269,117 @@ definition {
...
+        task ("Check it in its display page and access the edit") {
+		  ApplicationsMenu.gotoPortlet(
+			category = "Content",
+			panel = "Applications",
+			portlet = "Content Dashboard");
+
+			Click.clickNoMouseOver( locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED_MODIFIED_DATE");
+
+			Pause(locator1 = "5000");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Don't use Pause in tests, if you need to wait, use the waits macros instead

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira, whenever you can review the tests of this PR, it seems some of them are failing in CI

**Comment by RitaSilveira** [comment]
Fixed

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:forward

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira , this PR is failing the SF test :( Can you review it please?

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira , the SF keeps failing :( Please review again

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
@RitaSilveira it seems there are some problems because your pull is behind master: "_Target exception: java.lang.RuntimeException: Unable to rebase RitaSilveira-1297-LPS-116840 to master
error: Failed to merge in the changes."_ 

Try to rebase master and send again

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
See https://github.com/liferay-tango/liferay-portal/pull/1380

**Pull request closed without merging**